main.go

```diff
@@ -3,14 +3,16 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/eldeal/skills/config"
+	"github.com/gorilla/mux"
 	"io/ioutil"
 	"log"
 	"net/http"
-
-	"github.com/gorilla/mux"
 )
 
 func main() {
+	// Get Config
+	cfg, _ := config.Get()
```

Comment:
Only comment from me is it's good practice to handle this error.  In this case I would say we should log a helpful error message to give the error some context and then exit the application - the app can't function correctly so there is no point in trying to go on.

Otherwise this all looks good.

Reply:
This makes a lot of sense - thank you. I've made the change